docs/usage/validators.md

```diff
@@ -159,12 +175,18 @@ except ValidationError as e:
     """
 ```
 
+Note that the type of validator dictates the order of validation, not the order in which validators are applied in an `Annotated` type or class definition. In the following example, `func2` will be called before `func1`.
```

Comment:
Need to clarify that within a given type they go from right to left and back. That is it goes from right to left running all before validators (or calling into wrap validators) and then left to right back out calling all after validators. Doesn’t need to be in this paragraph, it’s just that as it stands right now this says “the order in Annotated doesn’t matter” but it does.